packages/formstr-app/src/containers/FormFiller/QuestionNode/InputFiller.tsx

```diff
@@ -0,0 +1,35 @@
+import { AnswerSettings, AnswerTypes } from "@formstr/sdk/dist/interfaces";
+import { Input } from "antd";
+import TextArea from "antd/es/input/TextArea";
+import { ChangeEvent } from "react";
+
+interface InputFillerProps {
+  answerType: AnswerTypes;
+  answerSettings: AnswerSettings;
+  onChange: (answer: string, message?: string) => void;
+}
+
+export const InputFiller: React.FC<InputFillerProps> = ({
+  answerType,
+  answerSettings,
+  onChange,
+}) => {
+  const handleInputChange = (
+    e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    onChange(e.target.value);
+  };
+  const getInput = (
+    answerType: AnswerTypes,
+    answerSettings: AnswerSettings
+  ) => {
+    switch (answerType) {
```

Comment:
instead of using switch, use object map where key will be answer type and value will be function according to answer type